drivers/volume/portworx/portworx.go

```diff
@@ -222,6 +227,16 @@ type statusJSON struct {
 	Cmd    string
 }
 
+func (d *portworx) executeCommandToPod(podCmd, podName, podNamespace string) (string, error) {
```

Comment:
This is a redundant method, there is an existing method in node driver to execute command inside pods, example of it's usage:
https://github.com/portworx/torpedo/blob/99b2904b4de9eacc906a3439084d8eb880c015a6/tests/basic/sharedv4_svc_test.go#L1713

Reply:
done.